docs/ops/detection/ExperimentalDetectronGenerateProposalsSingleImage_8.md

```diff
@@ -0,0 +1,156 @@
+filled with zeroes. If *dynamic_output* is false, then need not to fill output tensors with zeros. No matter attribute
+*dynamic_output* is true or false, the value will be the actual number of proposals.
+
+**Attributes**:
+
+* *min_size*
+
+    * **Description**: The *min_size* attribute specifies minimum box width and height.
+    * **Range of values**: non-negative floating-point number
+    * **Type**: float
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *nms_threshold*
+
+    * **Description**: The *nms_threshold* attribute specifies threshold to be used in the NMS stage.
+    * **Range of values**: non-negative floating-point number
+    * **Type**: float
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *pre_nms_count*
+
+    * **Description**: The *pre_nms_count* attribute specifies number of top-n proposals before NMS.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *post_nms_count*
+
+    * **Description**: The *post_nms_count* attribute specifies number of top-n proposals after NMS.
+    * **Range of values**: non-negative integer number
+    * **Type**: int
+    * **Default value**: None
+    * **Required**: *yes*
+
+* *coordinates_offset*
+
+    * **Description**: The *coordinates_offset* attribute specifies the relationship between ROI location points and ROI width and height. For example if *coordinates_offset* is false, the `width = x_right - x_left`. If *coordinates_offset* is true, the `width = x_right - x_left + 1`.
+    * **Range of values**: true or false
+    * **Type**: boolean
+    * **Default value**: false
+    * **Required**: *no*
+
+* *nms_eta*
+
+    * **Description**: eta parameter for adaptive NMS.
+    * **Range of values**: a floating-point number in close range `[0, 1.0]`.
+    * **Type**: float
+    * **Default value**: `1.0`
+    * **Required**: *no*
+
+* *dynamic_output*
+
+    * **Description**: The *dynamic_output* attribute specifies whether the output ROIs and ROIs scores are dynamic.
+    * **Range of values**: true or false
+    * **Type**: boolean
+    * **Default value**: false
+    * **Required**: *no*
+
+
+**Inputs**
+
+* **1**: A 1D tensor of type *T* with 3 elements `[image_height, image_width, scale_height_and_width]` providing input
+image size info. **Required.**
+
+* **2**: A 2D tensor of type *T* with shape `[height * width * number_of_channels, 4]` providing anchors. **Required.**
+
+* **3**: A 3D tensor of type *T* with shape `[number_of_channels * 4, height, width]` providing deltas for anchors.
+Height and width for third, fourth and fifth inputs should be equal. **Required.**
+
+* **4**: A 3D tensor of type *T* with shape `[number_of_channels, height, width]` providing proposals scores.
+**Required.**
+
+* **5**: A 3D tensor of type *T* with shape `[number_of_channels * 4, height, width]` providing variances for anchors.
```

Comment:
Also, I have question here - what is "number_of_channels"? According to Paddle, it is "A", that is number of anchors.

Reply:
yes, it's "A". the name is copied from ExperimentalDetectronGenerateProposalsSingleImage_6.md